src/data-editor/data-editor.tsx

```diff
@@ -179,6 +179,9 @@ const DataEditor: React.FunctionComponent<DataEditorProps> = p => {
     );
 
     const mangledCols = React.useMemo(() => {
+        // Clear the grid selection to handle the case where changing columns
+        // can highlight new unexpected selections
+        setGridSelection(undefined);
```

Comment:
Woah woah woah, careful with this. The function inside `useMemo` runs at render time. We shouldn't have side effects on `useMemo`.

Reply:
Removed this for now